packages/frontend/app/components/pod/contacts/contact-table/index.hbs

```diff
@@ -16,6 +17,13 @@
         <td>
           {{first-8 contact.publicKeyAsHex}}...
         </td>
+        <td>
+          <a class='button' 
```

Comment:
This may be simpler?

```hbs
<LinkTo 
  class='button'
  @route='contacts.verify-friend' 
  @model={{contact.id}} 
>
  {{t contact.verification }}
</LinkTo>
```

Reply:
docs here, if you're curious: https://guides.emberjs.com/release/routing/linking-between-routes/

Reply:
I agree it is cleaner! I tried using it but I had my route set up incorrectly